app/uk/gov/hmrc/plasticpackagingtax/registration/forms/FullName.scala

```diff
@@ -20,30 +20,21 @@ import play.api.data.Forms.text
 import play.api.data.{Form, Forms}
 import play.api.libs.json.{Json, OFormat}
 
-case class FullName(firstName: String, lastName: String)
+case class FullName(value: String)
```

Comment:
Do we still need this `FullName` class or should we refactor to use a String?

Reply:
This is the "form" class which does the binding from the view, so yes.  It is now analogous to`case class JobTitle(value: String)` or `case class EmailAddress(value: String)`